canvas_modules/common-canvas/src/common-canvas/cc-right-flyout.jsx

```diff
@@ -35,7 +84,12 @@ class CommonCanvasRightFlyout extends React.Component {
 				? "right-flyout-panel under-toolbar"
 				: "right-flyout-panel";
 			rightFlyout = (
-				<div className={rfClass}>
+				<div className={rfClass} id="right-flyout-panel" ref={ (ref) => (this.commonCanvasRightFlyout = ref) }>
```

Comment:
This shouldn't have an id of `right-flyout-panel`.  We should try not to use ids in common-canvas or common-properties.  If we do, then they need to be unique for that canvas since we can have multiple instances in the DOM.

Reply:
Ok, I have removed this now.